scripts/upload/add_csu_gpr_AK.py

```diff
@@ -7,59 +7,41 @@
 
 """
 
+from os.path import abspath, expanduser
 from pathlib import Path
-from snowexsql.db import get_db
-from snowex_db.upload import PointDataCSV
+from snowexsql.db import db_session_with_credentials
+from snowex_db.upload.points import PointDataCSV
 import pandas as pd
 
 
+# Issue #71
+
 def main():
     file = Path('../download/data/SnowEx223_FLCF_1GHz_GPR_CSU.csv')
 
-    # Fix quirks
-    df = pd.read_csv(file, dtype=str)
-
-    # Upload is not able to handle the Notes col. So just remove it for now
-    modified = file.parent.joinpath(file.stem + f'_mod{file.suffix}')
-    print(f"Removing Notes Column prior to upload. Writing to {modified}")
-
-    coi = [c for c in df.columns if c != 'Notes']
-
-    # No time is a problem. Use 12 AKST == 9pm (21:00) UTC
-    df['Time[HHMM]'] = '21:00'
-
-    # Write out the modified version
-    df[coi].to_csv(modified, index=False)
```

Comment:
Notice I ditched the modified function here - The tests work fine without doing this